README.md

```diff
@@ -1,2 +1,7 @@
+![hourly](https://github.com/enarx/bot/workflows/hourly/badge.svg)
+![cleanup-sprint-board](https://github.com/enarx/bot/workflows/cleanup-sprint-board/badge.svg)
+[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/enarx/bot.svg)](http://isitmaintained.com/project/enarx/bot "Average time to resolve an issue")
+[![Percentage of issues still open](http://isitmaintained.com/badge/open/enarx/bot.svg)](http://isitmaintained.com/project/enarx/bot "Percentage of issues still open")
```

Comment:
I'm not opposed to merging these, but it's worth noting that we haven't really been using issues in a high-throughput way like some of Enarx's other repos. They've mostly been used to denote larger tasks that get done in a few weeks. As such, the numbers on these badges look way worse than I believe they are.